build.sh

```diff
@@ -1,33 +1,30 @@
 #!/bin/sh -ex
 
+rm -rf ./build
+cp package.json generator
+
 # Prepare the generated source directory
 rm -rf ./generated-src
 mkdir -p generated-src
 cp generator/http.ts generated-src
 
 # Compile and run the generator
 tsc -p tsconfig.json
-node ./build/generate.js
-
-rm -rf ./docs
-typedoc --target ES5 --out ./docs/ --readme README.md generated-src
-touch ./docs/.nojekyll
+node --experimental-json-modules ./build/generate.js
```

Comment:
```
node.exe: --experimental-json-modules requires --experimental-modules be enabled
```
maybe due to my version?

Reply:
The `type: module` thing in package.json automatically implies `--experimental-modules`